apps/crn-server/src/controllers/research-tags.ts

```diff
@@ -24,4 +30,12 @@ export default class ResearchTags implements ResearchTagController {
       filter,
     });
   }
+
+  async fetchAll(options: FetchAllOptions): Promise<ListResearchTagResponse> {
```

Comment:
for this controller is there any point for leaving the `fetch` method as is? i thought we would just put the fetch-all implementation there